.env.example

```
@@ -3,4 +3,5 @@ VITE_FIREBASE_AUTH_DOMAIN=
 VITE_FIREBASE_PROJECT_ID=
 VITE_FIREBASE_STORAGE_BUCKET=
 VITE_FIREBASE_MESSAGING_SENDER_ID=
-VITE_FIREBASE_APP_ID=
\ No newline at end of file
+VITE_FIREBASE_APP_ID=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_FIREBASE_APP_ID key should go before the VITE_FIREBASE_AUTH_DOMAIN key

(UnorderedKey)